### ISSUE TITLE
OneStop Service Metadata

### ISSUE THREAD

**Issue by wendy144** [cedardevs/feedback#3]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by wendy144** [comment]
OneStop Service Metadata requirement suggestion:
Please provide ISO (for Collection-level) and ISO-Lite (for Granular-level) metadata examples which depict how to properly include Service Metadata.

**Comment by forgo** [comment]
@StrayCat1 Do we need to elaborate our documentation for our recent changes to services?

See: [https://github.com/cedardevs/onestop/blob/master/docs/design/iso-indexing-mapping.md#services](https://github.com/cedardevs/onestop/blob/master/docs/design/iso-indexing-mapping.md#services)

**Comment by caesterlein** [comment]
Yes, that will need to be updated. I will hold off until those changes are on master.
